southbound/impl/src/test/java/org/opendaylight/ansible/southbound/AnsibleCommandServiceImplTest.java

```diff
@@ -23,6 +23,10 @@
 import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.RunAnsibleCommandInput;
 import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.RunAnsibleCommandInputBuilder;
 import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.RunAnsibleCommandOutput;
+import org.opendaylight.yang.gen.v1.urn.opendaylight.ansible.command.rev180821.run.ansible.command.input.command.type
```

Comment:
this is OK to be longer than 120 and should be, otherwise checkstyle will complain